### DISCUSSION TITLE
Feature Request: Ability to annotate a single bar

### DISCUSSION THREAD

**Discussion by RvBurik** [chartjs/chartjs-plugin-annotation#484] [Q&A] [answered]
The possibility to draw a line on specific bars. This feature was already implemented in `v2` ([here](https://github.com/chartjs/chartjs-plugin-annotation/commit/be31a56f3ab1682d93ac84531bfd6528aa19302f)), by setting the `onlyForDataIndex` field. This worked on both horizontal and vertical bars. Unfortunately, this feature is currently not available in the newer version of this plugin. It would be great to see it back in `v3`!

Version in screenshot: 
- ChartJS: 2.9.3
- ChartJS-plugin-annotation: 0.5.7

![image](https://user-images.githubusercontent.com/18439206/125470268-1db75896-6e60-4e50-87fe-14282857b492.png)
![image](https://user-images.githubusercontent.com/18439206/125470468-8f82c084-02be-4283-b176-9db7c5011fba.png)

**Comment by kurkle** [comment, marked as answer]
Already possible, though you'll need to manually adjust for categoryPercentage, barPercentage and number of bars per category

https://codepen.io/kurkle/pen/XWgWYab
![image](https://user-images.githubusercontent.com/27971921/130584848-9816dfae-8d60-4d9f-b97b-3f00483041e4.png)


https://codepen.io/kurkle/pen/xxrxzJg
![image](https://user-images.githubusercontent.com/27971921/130584759-a76d4c2e-632e-48f4-9788-eab21fffb20d.png)
